_inc/client/plans/single-product-backup.jsx

```diff
@@ -179,52 +288,20 @@ class SingleProductBackupBody extends React.Component {
 		} );
 	};
 
-	handleLandingPageLinkClick = selectedBackupType => () => {
-		analytics.tracks.recordJetpackClick( {
-			target: 'landing-page-link',
-			feature: 'single-product-backup',
-			extra: selectedBackupType,
-		} );
-	};
-
 	render() {
-		const {
-			backupOptions,
-			billingTimeFrame,
-			currencyCode,
-			selectedBackupType,
-			upgradeLinks,
-		} = this.props;
+		const { backupOptions, billingTimeFrame, currencyCode, selectedBackupType } = this.props;
 
-		// TODO: Move out those somewhere else to make this a flexible and fully reusable component.
-		const upgradeTitles = {
-			'real-time': __( 'Upgrade to Real-Time Backups' ),
-			daily: __( 'Upgrade to Daily Backups' ),
-		};
+		const selectedBackup = get( backupOptions, selectedBackupType, null );
 
 		return (
 			<React.Fragment>
-				<p>
-					{ __(
```

Comment:
I see we removed this link, but I expected that we move it to the subheader, as shown in Figma.

Reply:
So, the `onClick` handler here uses the currently selected plan, which is data only available in this component. If we'd like to add this link to the header, we'd need to not have the analytics click event called in `handleLandingPageLinkClick`. Is that acceptable?

Reply:
I think that to overcome this limitation we could keep the `selectedBackupType` value in the parent component's (`ProductSelector`) state where the header is defined.

Reply:
I've actually went ahead and issued a PR that does just that - moves the link to the parent component: #14229 

Please, review and - depending which PR gets merged first - rebase the PR. I'm happy to take care of that on Monday as well.